content/videos/challenges/167-prime-spiral/showcase/contribution-1708894992.json

```
@@ -0,0 +1,9 @@
+{
+  "title": "Zooming Out Animation of the Ulam Spiral",
+  "author": {
+    "name": "Polo Clebicar",
+    "url": "https://github.com/PoloClebicar"
+  },
+  "url": "https://github.com/PoloClebicar/Ulam_Spiral/tree/main/p5",
```

```suggestion
  "url": "https://github.com/PoloClebicar/Ulam_Spiral",
```

This link 404'd so I think it's beter to use the main repo, is that ok with you @PoloClebicar? Thank you for submitting!

---

No problem. I am pretty new to GitHub, is this change to be made on my part?
I also copied  this code to a Saved Sketch on the p5 website on an account linked tot his one, and submitted that, maybe that easier. If I need to change anything let me now.